### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Thoth1111**
In this branch, I styled my bookstore app and achieved the following objectives in my commits:
✅ Added missing JSX elements without adding extra functionality
✅ Matched my design to the Zeplin's design.
✅ Used CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -15,28 +18,44 @@ const Book = () => {
...
+              <hr className="vertical line-1" />
+              <button type="button" className="btn-type1">Edit</button>
+            </div>
+          </div>
+          <div className="progress-cont">
+            <CircularProgressbar value={66} />
+          </div>
+          <div>
+            <span className="percentage">64%</span>
+            <br />
```
Comment (AnselemOdims) [inline src/components/Book.js]: - [x] Good job on the styling but please ensure that the space between the progress bar and percentage matches that of the original design.

### Your Design
image

### Original Design
image

File: src/components/Book.js [hunk 2]
```diff
@@ -15,28 +18,44 @@ const Book = () => {
...
+            <span className="entry-completed">Completed</span>
+          </div>
+          <hr className="vertical line-2" />
+          <div>
+            <h4 className="entry-chapter">CURRENT CHAPTER</h4>
+            <h4 className="chapt-no">Chapter 17</h4>
+            <button type="button" className="btn-type2">UPDATE PROGRESS</button>
+          </div>
+        </section>
```
Comment (AnselemOdims) [inline src/components/Book.js:56]: - [x] Please ensure that this section of the book display has enough space to the right when compared to the original design.

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Thoth1111 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Book.js [hunk 3]
```diff
@@ -15,28 +18,44 @@ const Book = () => {
...
+                Remove
+              </button>
+              <hr className="vertical line-1" />
+              <button type="button" className="btn-type1">Edit</button>
+            </div>
+          </div>
+          <div className="progress-cont">
+            <CircularProgressbar />
+          </div>
+          <div className="stats">
```
Comment (DuaneDave) [inline src/components/Book.js:45]: Good job taking care of the spaces at the right 👍. However, while addressing that issue I  realized that you have added more issues, the progress bar now collides with the text and the buttons are now looking squished up. I perceive the resolution of your PC is quite big and the design is looking good on your side, so I am recommending you test it out with a resolution of width `1300px` and below which is my range.
I recommend that this issue be addressed in order to provide a consistent UI for users on different desktop screen sizes.


Here is what I see

bookstore

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Thoth1111 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
✔️ PR summary present
✔️ ReadMe is professional and descriptive
✔️ Good job attempting requested changes


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 3 above)
Comment (Thoth1111) [reply src/components/Book.js:45]: Thanks, I've made changes to make it work with a minimum desktop width of 1200px, so it should be fine now.

File: src/components/Navbar.js [hunk 4]
```diff
@@ -1,18 +1,26 @@
...
+import '../styles/nav.css';
 
 function Navigation() {
   return (
-    <div>
+    <nav>
       <ul>
         <li>
-          <Link to="/">Books</Link>
+          <h1>Bookstore CMS</h1>
```
Comment (huseyinbiyik) [inline src/components/Navbar.js:11]: [OPTIONAL] You did a great job with the design. But fonts does not match with the Zeplin design. Kindly fix them.
Screenshot 2023-03-04 at 8 13 37 PM
Screenshot 2023-03-04 at 8 14 06 PM

File: src/routes/Books.js [hunk 5]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import Form from '../components/Form';
 import Book from '../components/Book';
+import '../styles/books.css';
 
 const Books = () => (
```
Comment (huseyinbiyik) [inline src/routes/Books.js:6]: [OPTIONAL] Unfortunately, same issue occurs here. You have no added fonts so system chose the default font family. I highly recommend you to add the related fonts to match Zeplin design.
Screenshot 2023-03-04 at 8 13 52 PM
Screenshot 2023-03-04 at 8 14 27 PM

**Review by huseyinbiyik — APPROVED** [review]
Hi @Thoth1111

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Gitflow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Books.js (hunk 5 above)
Comment (Thoth1111) [reply src/routes/Books.js:6]: Thank you. I will open an issue and address these optional changes after the merge. I appreciate the approval 🙏

**Pull request merged**